inventory/views.py

```diff
@@ -161,13 +161,14 @@ def loan_return_post(request):
 
 @api_view(["POST"])
 @permission_classes([IsAuthenticated])
-def revert_order(request, order_id):
+def revert_order(request):
+    order_id = request.data
     try:
         if order_id is None:
             return Response({"error": "Invalid request body"}, status=400)
         try:
             instance = Order.objects.get(pk=order_id)
-            # TODO: handle delete
+            manage_items_return(instance)
```

Comment:
hmm `manage_items_return` seems a little misleading, perhaps we can rename it to `handle_delete` or `revert_order_items`